modules/installation-configuration-parameters.adoc

```diff
@@ -1096,13 +1105,33 @@ Additional GCP configuration parameters are described in the following table:
 |Any valid region name, such as `us-central1`.
 
 |`platform.gcp.controlPlaneSubnet`
-|The name of the existing subnet in your VPC that you want to deploy your control plane machines to.
+|The name of the existing subnet where you want to deploy your control plane machines.
 |The subnet name.
 
 |`platform.gcp.computeSubnet`
-|The name of the existing subnet in your VPC that you want to deploy your compute machines to.
+|The name of the existing subnet where you want to deploy your compute machines.
 |The subnet name.
 
+|`platform.gcp.createFirewallRules`
+|Optional. Set this value to `Disabled` if you want to create and manage your firewall rules through the use of network tags. By default, the cluster will automatically create and manage the firewall rules that are required for cluster communication. Your service account must have `roles/compute.networkAdmin` and `roles/compute.securityAdmin` privileges in the host project to perform these tasks automatically. If your service account does not have the `roles/dns.admin` privilege in the host project, it must have the `dns.networks.bindPrivateDNSZone` permission.
+|`Enabled` or `Disabled`. The default value is `Enabled`.
+
+|`platform.gcp.publicDNSZone.project`
+|Optional. The name of the project which contains the public DNS zone. If you set this value, your service account must have the `roles/dns.admin` privilege in the specified project. If you do not set this value, it defaults to `gcp.projectId`.
+|The name of the project that contains the public DNS zone.
+
+|`platform.gcp.publicDNSZone.id`
+|Optional. The ID or name of an existing public DNS zone.  The public DNS zone domain must match the `baseDomain` parameter. If you do not set this value, the installation program will create a public DNS zone in the service project.
```

Comment:
`If you do not set this value, the installation program will create a public DNS zone in the service project.` 

should be: 

`If you do not set this value, the installation program will use the public DNS zone,  in the service project, corresponding to the baseDomain.`

Reply:
<img width="787" alt="Screen Shot 2022-11-07 at 1 25 57 PM" src="https://user-images.githubusercontent.com/93731574/200386513-08691029-2e60-460c-9231-eab770d2fe43.png">
@patrickdillon requested that I change it to "will create" instead of "will use". Patrick can you advise as to which phrasing is correct here? Thank you

Reply:
@jianli-wei is correct. For the public zone it should be "will use", for the private zone it should be "will create." 

Nice catch jianli!

Reply:
Thanks @patrickdillon and @jianli-wei, I have updated the PR to reflect these changes. If this looks good I will proceed with peer review and merge review.